codegenerator/cli/src/executor/init.rs

```diff
@@ -27,7 +27,7 @@ use std::{env, path::PathBuf};
 fn is_valid_release_version_number(version: &str) -> bool {
     let re_version_pattern = Regex::new(r"^\d+\.\d+\.\d+(-rc\.\d+)?$")
         .expect("version regex pattern should be valid regex");
-    re_version_pattern.is_match(version) || version.starts_with("0.0.0-main-")
```

Comment:
Now starts with 2.0.0 so just checking that it contains "-main"

Reply:
Interesting. So this code actually won't, for example, match on release. It will only match on releases or those with RC on them?
 
 Eg this won't match: `2.0.0-release-concurrent-partitions-20241004091855-c02f47f9`

Reply:
This is just for contract import, I think we can improve it, it was a bit of a hack https://github.com/enviodev/hyperindex/pull/244